src/components/Employee/Deductions/Deductions.tsx

```diff
@@ -139,7 +139,12 @@ export const Root = ({ employeeId, className }: DeductionsProps) => {
       //Deletion of deduction is simply updating it with active: false
       const updateMutationResponse = await updateDeductionMutation.mutateAsync({
         garnishment_id: payload.uuid,
-        body: { ...payload, active: false, version: payload.version as string },
+        body: {
+          ...payload,
+          total_amount: payload.total_amount ?? undefined,
```

Comment:
After updating openapi, total_amount in the body here needed to be string or undefined but it was optionally null from the payload. Update to force it to undefined if null to satisfy the type